CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
+###
```

Comment:
Can you make this header `Fixed` like the one below.

Reply:
Sorry, missed this. Fixed in updated PR.